### DISCUSSION TITLE
Inlined source maps add hidden size, causing confusion

### DISCUSSION THREAD

**Discussion by msafi** [webpack-contrib/webpack-bundle-analyzer#582] [Ideas]
In my webpack config I had `devtool` set to `inline-source-map`. So the inlined source maps were included in the final `bundle.js`. I didn't know that, so when Webpack Bundle Analyzer was telling me that the highest level code container in my bundle is 500KB but right above it the bundle.js size is 2MB, I was very confused.

It would be useful to help the user know what's going on in these cases.

**Comment by valscion** [comment]
This is similar to https://github.com/webpack-contrib/webpack-bundle-analyzer/issues/235
